ts/components/dialog/SessionModal.tsx

```diff
@@ -27,6 +27,7 @@ interface State {
   isVisible: boolean;
 }
 
+// NOTE This is currently unused.
```

Comment:
should we keep it then?

Reply:
So I know that iOS has a caption editor and I suspect android does too. So at some point in the future we may need this in order to be "consistent with other platforms".

Reply:
kill it !

Reply:
it rose back from the dead! 🧟